Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about scuba diving?"
+    m 3eub "after immersion in water, you can see a wonderful underwater world full of diverse marine life."
+    m 7eub "There you will meet:schools of fish, coral reefs, and sea greens,jellyfish's."
```

Comment:
```suggestion
    m 7eub "There, you can see schools of fish, coral reefs, jellyfish, and sea greens."
```
I've never been scuba diving, but I don't think it's a guarantee that you'd see all those things.